dynamic_rest/filters.py

```diff
@@ -665,4 +669,8 @@ def get_valid_fields(self, queryset, view):
                 if not getattr(field, 'write_only', False) and
                 not field.source == '*' and field_name in valid_fields
             ]
+
+        if 'get_template_context' == caller:
```

Comment:
This seems hacky / brittle -- why should the caller change the return value? Is there any other way we can retain compatibility that is caller-agnostic?